apps/frontend/src/app/0_meta/DeferedDataLoader/withDeferedRouterData.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Await, useLoaderData } from 'react-router-dom';
+
+import { ErrorRenderer } from '../../1_atoms/ErrorRenderer/ErrorRenderer';
+import { LoaderWithLogo } from '../../1_atoms/LoaderWithLogo/LoaderWithLogo';
+
+export const withDeferedLoaderData = <
+  T extends object,
+  C extends { defered: T },
```

Comment:
```suggestion
export const withDeferredLoaderData = <
  T extends object,
  C extends { deferred: T },
```
Sorry to make you change this as its in multiple files, but "deferred" has two r's in it